tools/test-lib/react-app/VizComponent.js

```diff
@@ -6,9 +6,9 @@ function VizComponent({ data }) {
     <div style={{ height: `90vh`, width: `100%` }}>
```

Comment:
[Nit] Better to rename the file to follow kebab-case as other files we have. Thanks !